assets/alice.csv

```diff
@@ -511,50 +511,50 @@ reform_effect_desc;When activated:;;;;;;;;;;;;x
 special_rules;Special rules:;;;;;;;;;;;;x
 voting_rules;Voting rules:;;;;;;;;;;;;x
 hide_decision;Hide the notification for this decision;;;;;;;;;;;;x
-sup_point_gain;Monthly suppression point increase: �G+$val$;;;;;;;;;;;;x
+sup_point_gain;Monthly suppression point increase: �G+$val$;;;;;;;;;;;;x
```

Comment:
Lines like this are places where a utf 8 conversion has been performed that may look the same, but will probably break the text

Reply:
No way of fixing or avoiding it i think, ive only edited the csv file with codium, instead of vim, so not sure

Reply:
Well, you should roll back to an earlier version of the file then. And you will need to find an editor that doesn't perform automatic conversion. I don't edit the file in Visual studio for that reason